src/bioregistry/data/bioregistry.json

```diff
@@ -52774,6 +52774,37 @@
       }
     ]
   },
+  "gpml": {
```

Comment:
@jmillanacosta @egonw what do you think about using `wikipathways.gpml` instead?